.eslintrc.yaml

```diff
@@ -357,3 +358,27 @@ rules:
 
   react-hooks/exhaustive-deps: error
   react-hooks/rules-of-hooks: error
+
+  ####################################
+  # eslint-plugin-simple-import-sort #
+  ####################################
+
+  simple-import-sort/imports: [warn, {groups: [
+      # Node.js builtins. You could also generate this regex if you use a `.js` config.
+      # For example: `^(${require("module").builtinModules.join("|")})(/|$)`
+      ["^(assert|buffer|child_process|cluster|console|constants|crypto|dgram|dns|domain|events|fs|http|https|module|net|os|path|punycode|querystring|readline|repl|stream|string_decoder|sys|timers|tls|tty|url|util|vm|zlib|freelist|v8|process|async_hooks|http2|perf_hooks)(/.*|$)"],
+      # Packages. `react` related packages come first.
+      ["^react", "^@?\\w"],
```

Comment:
Not sure I see any reason to sort react stuff first.  I've always sorted it alphabetically with all the other package imports.